.github/workflows/callable-republish-package.yaml

```diff
@@ -0,0 +1,97 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+name: Callable-Republish-Package
+
+on:
+  workflow_call:
+    inputs:
+      runsOn:
+        default: ubuntu-latest
+        type: string
+      flavor:
+        required: true
+        type: string
+      options:
+        type: string
+        description: Additional options to set when running the test
+        default: ""
+      reports-path:
+        type: string
+        description: The path to test outputs to upload
+      reports-name:
+        type: string
+        description: The name prefix for upload test reports
+        default: playwright-report
+      timeout:
+        type: number
+        default: 40
+
+# Permissions for the GITHUB_TOKEN used by the workflow.
+permissions:
+  contents: read # Allows reading the content of the repository.
+  packages: read # Allows reading the content of the repository's packages.
+  id-token: write
+
+jobs:
+  run:
+    runs-on: ${{ inputs.runsOn }}
+    timeout-minutes: ${{ inputs.timeout }}
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Install UDS CLI
+        uses: defenseunicorns/setup-uds@ab842abcad1f7a3305c2538e3dd1950d0daacfa5 # v1.0.1
+        with:
+          # renovate: datasource=github-tags depName=defenseunicorns/uds-cli versioning=semver
+          version: v0.25.0
+
+      - name: Environment setup
+        run: |
+            uds run actions:setup-environment \
+            --set REGISTRY1_USERNAME="${{ secrets.IRON_BANK_ROBOT_USERNAME }}" \
+            --set REGISTRY1_PASSWORD="${{ secrets.IRON_BANK_ROBOT_PASSWORD }}" \
+            --set GH_TOKEN="${{ secrets.GITHUB_TOKEN }}" \
+            --set CHAINGUARD_IDENTITY="${{ secrets.CHAINGUARD_IDENTITY }}"
+        shell: bash
+
+      - name: Republish the package
+        run: |
+          uds run republish-package --set USE_CHECKPOINT=false --set FLAVOR=${{ inputs.flavor }} --no-progress ${{ inputs.options }}
```

Comment:
Why disable checkpoint?

Reply:
Talked to wayne, this is so we have a "full" test before publish in case checkpoint is being weird